### PR TITLE
Minor typos and suggestions to Packaging.md

### PR DESCRIPTION
**Pull request by maroneze**

### CODE REVIEW

File: doc/pages/Packaging.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ remove: ["ocamlfind" "remove" "project"]
 depends: "ocamlfind"
 ```
 
-The `opam-version`, `maintainer` and `version` fields are mandatory ; you should
+The `opam-version`, `maintainer` and `version` fields are mandatory; you should
```
Comment (dbuenzli) [inline doc/pages/Packaging.md]: Actually the `version` field should not be specified. You may want to correct this.

**Comment by maroneze** [comment]
Thank you @dbuenzli, updated.

File: doc/pages/Packaging.md [hunk 2]
```diff
@@ -130,7 +130,8 @@ than a single element: `maintainer`, `author`, `homepage`, `bug-reports`,
 `license` and `depends`.
 
 For the list of available fields and specification of the format, see the
-[OPAM manual](Manual.html#opam).
+[OPAM manual](Manual.html#opam). Also consider using the `opam lint` command to
+check the validity and quality of your `opam` file.
```
Comment (AltGr) [inline doc/pages/Packaging.md]: This is indeed very worth mentioning! Note also that `opam pin edit` runs a lint check already.

**Comment by AltGr** [comment]
Thanks!

**Pull request merged**